pkg/scheduler/scheduler.go

```diff
@@ -489,6 +437,11 @@ func newScheduler(
 		percentageOfNodesToScore: percentageOfNodesToScore,
 	}
 	sched.SchedulePod = sched.schedulePod
+	if failureHandler != nil {
+		sched.FailureHandler = failureHandler
```

Comment:
non-nil case is mostly called in UTs where may compose a fake FailureHandler function.

Reply:
Good point. Updated.